packages/ui/src/__tests__/atoms/Button.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
```

Comment:
You don't need to import React, describe, expect or it, just only vi from 'vitest'.

Reply:
Done